test/functional/core/remote_spec.lua

```diff
@@ -55,6 +55,8 @@ describe('Remote', function()
     end
 
     it('edit a single file', function()
+      if helpers.pending_freebsd(pending) then return end
```

Comment:
```suggestion
      if helpers.skip_fragile(pending) then return end
```

p.s. (separate commit/PR?) we should probably rename `pending_win32` to `skip_win32`

Reply:
Nah, let's do it, I like skip better.